From 20989a120fde70578711821e53a13b071c020193 Mon Sep 17 00:00:00 2001
From: "jing.zhang" <jing.zhang@amlogic.com>
Date: Fri, 7 Jan 2022 16:16:54 +0800
Subject: [PATCH] audio: Irdeto project request remove log [1/1]

PD#SWPL-67110

Problem:
Very verbose logs when zap to a service

Solution:
remove log

Verify:
rdk212

Change-Id: I8f2673f0670dd94576e4d67932399e6cf0d745fa
Signed-off-by: jing.zhang <jing.zhang@amlogic.com>
---
 dtv_audio_utils/dmxwrap/MultiHwDemux/AmHwMultiDemuxWrapper.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dtv_audio_utils/dmxwrap/MultiHwDemux/AmHwMultiDemuxWrapper.cpp b/dtv_audio_utils/dmxwrap/MultiHwDemux/AmHwMultiDemuxWrapper.cpp
index 7f642929..98e473b1 100644
--- a/dtv_audio_utils/dmxwrap/MultiHwDemux/AmHwMultiDemuxWrapper.cpp
+++ b/dtv_audio_utils/dmxwrap/MultiHwDemux/AmHwMultiDemuxWrapper.cpp
@@ -66,7 +66,6 @@ static void getAudioEsData(AmHwMultiDemuxWrapper* mDemuxWrapper, int fid, const
         mEsData->pts = es_header->pts;
         mDemuxWrapper->last_queue_es_apts = es_header->pts;
         mEsData->used_size = 0;
-        ALOGV("getAudioEsData %d mEsData->size %d mEsData->pts %lld",len,mEsData->size,mEsData->pts);
         dump_demux_data((void *)data_es, es_header->len, DEMUX_AUDIO_DUMP_PATH);
     } else {
         ALOGI("error es data len %d es_header->len %d",len, es_header->len);
-- 
2.31.0

